.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+## How to contribute to Open Bastion
+
+#### **Did you find a bug?**
+
+* You can either just create an issue on the GitHub repository or create a pull
+  request with your fix proposal. We will be happy to review and integrate it
+  if possible.
+* Do not forget to check if an existing issue is not already opened.
+* If possible, use the issue template.
+
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+* Ensure the PR description clearly describes the problem and the solution.
+* Add the corresponding issue number in your commit(s).
+
+#### **Did you fix whitesapce, format code, or make a purely cosmetic patch?**
+
+These kinds of fix are warmly welcomed (in contrast to rails for example) as
+long as they pass the tests. Also, it would be even better to improve the test
```

Comment:
Maybe something like : 

"Improving the tests to avoid these kind of fixes in the future is also greatly appreciated!"